Core/DISET/private/BaseClient.py

```diff
@@ -291,6 +294,16 @@ def __checkThreadID( self ):
 
   def _connect( self ):
 
+    # Check if the useServerCertificate configuration changed
+    if gConfig.useServerCertificate() != self.useCertificates:
```

Comment:
`gConfig.useServerCertificate()` returns either `True` or `False`. So, we will always enter here. Is this what you want?